xbmc/windows/GUIMediaWindow.cpp

```diff
@@ -565,7 +567,9 @@ void CGUIMediaWindow::ClearFileItems()
   m_unfilteredItems->Clear();
 }
 
-// \brief Sorts Fileitems based on the sort method and sort oder provided by guiViewState
+/*!
+ * \brief Sorts Fileitems based on the sort method and sort oder provided by guiViewState
```

Comment:
Becomes changes, also do I move this on GUIMediaWindow.cpp to a separate request